pkg/tcpip/network/ip_test.go

```diff
@@ -243,28 +259,32 @@ func TestIPv4Receive(t *testing.T) {
 		t.Fatalf("NewEndpoint failed: %v", err)
 	}
 
-	totalLen := header.IPv4MinimumSize + 30
+	ihl := header.IPv4MinimumSize + len(options)
+	totalLen := ihl + 30
```

Comment:
why 30?

Reply:
totalLen := header.IPv4MinimumSize + 30

Previously they used to add 30 with IPv4MinimumSize for the total length, we are not disturbing that test.